classes/Kohana/Asset/Collection.php

```diff
@@ -38,6 +38,30 @@
 	 * @var  int  last modified time
 	 */
 	protected $_last_modified = NULL;
+	
+	/**
+	 * @var bool show merged files paths
+	 */
+	protected $_show_paths = TRUE;
+	
+	/**
+	 * Setter / Getter for merged files path displaying
+	 * @author Piotr Gołasz <pgolasz@gmail.com>
+	 * @param bool/null $path
+	 * @return bool
+	 */
+	public function display_paths($path = NULL)
+	{
+		if(is_null($path))
```

Comment:
Could you put a space between `if` and the opening parenthesis? Same thing a couple places below. Thanks!